communitymanager/src/main/java/org/pesmypetcare/communitymanager/datacontainers/TagData.java

```diff
@@ -0,0 +1,33 @@
+package org.pesmypetcare.communitymanager.datacontainers;
+
+import androidx.annotation.NonNull;
+
+import java.util.List;
+
+/**
+ * @author Santiago Del Rey
+ */
+public class TagData {
+    private List<String> groups;
+    private List<String> forums;
+
+    /**
+     * Default constructor.
+     */
+    public TagData() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unnecessary constructors - the compiler will generate these for you](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5538337)